srv/components/s3server/init.sls

```diff
@@ -19,7 +19,7 @@
 include:
   - components.s3server.prepare
   - components.s3server.install
-  - components.s3server.config
+  - components.s3server.config.init_mod
```

Comment:
Should be components.s3server.config, no?
Also, where is srv/components/s3server/config/init.sls that includes the 3 config sub-modules?
We need it until we make the switch to CES workflow

Reply:
similar to above implementation